packages/discord.js/src/structures/SelectMenuBuilder.js

```diff
@@ -21,18 +21,31 @@ class SelectMenuBuilder extends BuildersSelectMenu {
     );
   }
 
+  /**
+   * Normalizes a select menu option emoji
+   * @param {SelectMenuOptionData|JSONEncodable<APISelectMenuOption>} selectMenuOption The option to normalize
+   * @returns {Array<SelectMenuOptionBuilder | APISelectMenuOption>}
```

Comment:
For consistency in the documentation:

```suggestion
   * @returns {Array<SelectMenuOptionBuilder|APISelectMenuOption>}
```

Reply:
`(SelectMenuOptionBuilder|APISelectMenuOption)[]` to be consistent with the majority of the documentation